### ISSUE TITLE
Travis-CI configuration not building and checking package? 

### ISSUE THREAD

**Issue by andybega** [ShanaScogin/BayesPostEst#45]
I don't think the Travis-CI configuration file actually ends with the package being built and checked. It checks the installed Stan version via `rstan::stan_version()` and declares success. 

There should be a log of the package install and R check results (this is from https://travis-ci.org/andybega/states):

<img width="1044" alt="Screen Shot 2019-08-30 at 21 53 59" src="https://user-images.githubusercontent.com/1353756/64044932-bbdb5500-cb70-11e9-9745-f0a55001c0fe.png">

There isn't.

<img width="1038" alt="Screen Shot 2019-08-30 at 21 55 16" src="https://user-images.githubusercontent.com/1353756/64045003-e5947c00-cb70-11e9-8aab-b73136c23814.png">

**Comment by andybega** [comment]
The `bug/travis` branch here has my history of failures so far https://travis-ci.com/andybega/BayesPostEst/builds

Installing the big packages like `rstan` and `rjags` seems to help, and I get up to the package build step when trying to knit the `get-started.Rmd` vignette. The code chunk trying to fit the first Stan model is error-ing out.

**Comment by ShanaScogin** [comment]
Hi Andy - I just pushed Rob's new feature and am getting caught up on this now. When did you fork? Can you grab the new stuff (incl the vignette) in master and see if you're having the same problems? I had a problem with stan a month and a half or so ago, but haven't had it since. I might be missing the issue you're talking about here though - I'll review it soon.

**Comment by andybega** [comment]
Hello, I forked a few days ago, the last commits I had from your side were 15 Aug. Updating now :). 

The issue here was that, if I understand right, the configuration file overrode the steps that are usually done to test an R package on Travis-CI, which are to build it and then R check on it. Instead, it was only installing rstan and then checking that the `stan_version()` command runs. Travis-CI didn't even get as far as trying to install `BayesPostEst`. 

Basically it wasn't really doing a proper test for `BayestPostEst`---I could have added in code that intentionally errors out and Travis-CI would still give a thumbs up.

**Comment by ShanaScogin** [comment]
Ah I see - the new versions shouldn't help that issue at all. Ya that's a bit of a problem. Thanks so much for looking into this. Is there something you need from me?

**Comment by andybega** [comment]
Not for this issue---the new travis.yml file in the pull request will fix the problem with Travis-CI. I guess if you are happy with the pull request the only thing would be for you to approve it.
